products/compute/terraform.yaml

```diff
@@ -1255,6 +1256,56 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           The Region in which the created address should reside.
           If it is not provided, the provider region is used.
 
+  PerInstanceConfig: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{project}}/{{zone}}/{{instance_group_manager}}/{{name}}"
+    mutex: networkEndpoint/{{project}}/{{zone}}/{{instance_group_manager}}
```

Comment:
it's gated against the network endpoint? if its just the IGM, maybe we should rename this mutex

Reply:
Ah, yeah copied from network endpoint and forgot to change